up42/auth.py

```diff
@@ -115,26 +139,29 @@ def _find_credentials(self) -> None:
         Sources the project credentials from a provided config file, error handling
         if no credentials are provided in arguments or config file.
         """
-        if self.project_id is None or self.project_api_key is None:
+        if self.credentials_id is None or self.credentials_key is None:
             if self.cfg_file is None:
-                raise ValueError("Provide project_id and project_api_key via arguments or config file!")
+                raise ValueError("Provide credentials via arguments or config file!")
 
             # Source credentials from config file.
             try:
-                with open(self.cfg_file) as src:
+                if self.auth_type == AuthType.PROJECT.value:
+                    logger.info(
+                        "Project based authentication will be deprecated soon."
+                        "Please follow the [authentication guidelines](https://sdk.up42.com/authentication/)."
+                    )
+                with open(self.cfg_file, encoding="utf-8") as src:
                     config = json.load(src)
                     try:
-                        self.project_id = config["project_id"]
-                        self.project_api_key = config["project_api_key"]
+                        self.credentials_id = config[self.cred_mapping[self.auth_type]["credentials_id"]]
+                        self.credentials_key = config[self.cred_mapping[self.auth_type]["credentials_key"]]
                     except KeyError as e:
-                        raise ValueError(
-                            "Provided config file does not contain project_id and " "project_api_key!"
-                        ) from e
+                        raise ValueError("Provided config file does not contain valid credentials.") from e
```

Comment:
We actually don't know here whether the credentials are valid or not.
All we know is the keys are wrong in the config file.
Consider re-phrasing the ValueError string.